test/shopify-cli/task/ensure_env_test.rb

```diff
@@ -14,18 +14,148 @@ def setup
         ShopifyCli::Tunnel.stubs(:start)
       end
 
-      def test_ask_strips_out_https_from_shop
-        CLI::UI.expects(:ask).times(3)
-          .returns('apikey', 'apisecret', 'https://test-shop.myshopify.com')
-        @task = EnsureEnv.call(@context)
-        assert_equal('test-shop.myshopify.com', Project.current.env.shop)
-      end
-
-      def test_writes_env_if_no_env
-        CLI::UI.expects(:ask).times(3)
-          .returns('apikey', 'apisecret', 'https://test-shop.myshopify.com')
-        @task = EnsureEnv.call(@context)
-        assert_equal('apikey', Project.current.env.api_key)
+      def test_create_new_app_if_none_available
+        response = [{
+          "id" => 421,
+          "businessName" => "one",
+          "stores" => [{
+            "shopDomain" => "store.myshopify.com",
+          }],
+          "apps" => [],
+        }]
+        ShopifyCli::PartnersAPI::Organizations.stubs(:fetch_with_app).returns(response)
```

Comment:
It's better to use `expects(:fetch_with_app).with(@context).returns(...)`. I know you probably just copied it from before, but it's still better to make improvements of the code when you see the opportunity.